third_party/glslang-spirv/SpvBuilder.cpp

```diff
@@ -196,6 +196,21 @@ Id Builder::makeRayQueryType()
     return type->getResultId();
 }
 
+Id Builder::makeHitObjectNVType()
+{
+    Instruction *type;
+    if (!hit_object_nv_type) {
```

Comment:
Braces on next line.

Reply:
The formatting in the file is so random that I wasn't sure which example I should follow. I'll make it match `makeRayQueryType` above, braces will be on the next line then (and braces for `else` below will be omitted).